src/canvas/SelectableCanvas.ts

```diff
@@ -796,62 +796,59 @@ export class SelectableCanvas<EventSpec extends CanvasEvents = CanvasEvents>
   }
 
   /**
-   * Internal Function used to search inside objects an object that contains pointer in bounding box or that contains pointerOnCanvas when painted
+   * Search for objects containing {@link pointer}.
+   * Search is not greedy, returning once a hit is found.
    * @param {Array} [objects] objects array to look into
-   * @param {Object} [pointer] x,y object of point coordinates we want to check.
-   * @return {FabricObject} **top most object from given `objects`** that contains pointer
-   * @private
+   * @param {Object} [pointer] point coordinates to check
+   * @param {boolean} [param2.searchStrategy] strategy
+   * @returns {FabricObject[]} path of hits starting from **top most** object on screen.
    */
-  _searchPossibleTargets(
+  protected findTargetsTraversal(
```

Comment:
What's the difference between this and `searchPossibleTargets`

Reply:
changed name to break itr cleanly